byterun/win32.c

```diff
@@ -914,3 +918,62 @@ void caml_restore_win32_terminal(void)
   if (startup_codepage != 0)
     SetConsoleOutputCP(startup_codepage);
 }
+
+/* Detect if a named pipe corresponds to MSYS/Cygwin tty: see
+
+   https://github.com/mirror/newlib-cygwin/blob/master/winsup/cygwin/dtable.cc#L932
+*/
+typedef
+BOOL (WINAPI *tGetFileInformationByHandleEx)(
+  HANDLE                    hFile,
+  FILE_INFO_BY_HANDLE_CLASS FileInformationClass,
+  LPVOID                    lpFileInformation,
+  DWORD                     dwBufferSize
+);
```

Comment:
Just for brevity, function pointer definitions don't need parameter names (cf. https://github.com/ocaml/ocaml/blob/trunk/otherlibs/win32unix/symlink.c#L31)